src/components/domain/user/UserSummaryCard.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import ShadowCard from '@/components/common/card/ShadowCard';
+import TagInput from '@/components/common/input/TagInput';
+import CirclePlanetIcon from './CirclePlanetIcon';
+import { ChevronRight } from 'lucide-react';
+
+export default function UserSummaryCard() {
+  const handleOpenUserProfile = () => {
+    console.log('프로필 보러가기 연결 예정');
+  };
+
+  // NOTE: project API 연동 이후 하드코딩된 값 변경 예정
+
+  return (
+    <ShadowCard className="flex h-[157px] w-[512px] flex-col justify-between p-6">
+      <div className="flex items-start space-x-4">
+        <CirclePlanetIcon className="bg-gray-100" />
+        <div className="flex flex-col justify-center">
+          <p className="body8">이지영</p>
+          <p className="text-gray-500 display5">fj298@gmail.com</p>
+        </div>
+      </div>
+      <div className="mt-auto flex items-center justify-between">
+        <div className="flex items-center">
```

Comment:
```suggestion
        <div className="flex items-center gap-2">
```
TagInput에 `mr-2` 를 빼고 부모 flex에 `gap` 을 지정해서 간격을 조정하는게 더 좋을 것 같습니다!
다음에 수정해주실때 참고 부탁드립니다!

Reply:
언급 주신 부분 한 번에 수정했습니다!
✅ [2163686](https://github.com/PRism-Works/PRism-FE/pull/67/commits/2163686c67a13a85f5db9160deafc88b0ac73b24)